src/tinypeel/tinypeel.py

```diff
@@ -17,7 +17,14 @@ def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
     # Right now maf _only_ uses the penetrance so can be estimated once.
     if args.est_alt_allele_prob:
         PeelingUpdates.updateMaf(pedigree, peelingInfo)
-
+    if args.alt_allele_prob_file is not None:
+        pedigree.readInAAP(pedigree, args.alt_allele_prob_file)
+        for ind in pedigree:
+            if ind.MetaFounder is not None:
```

Comment:
A condition of the individual to be the founder is also required (try `ind.isFounder()`). In the `tinyhouse.Pedigree`, I set metafounders for all of the individuals for simplicity, so the progenies of the progenies of different metafounders all share a default metafounder, so we don't want to update their anterior probabilities otherwise it would interfere with the information while passing down the pedigree if the alternative allele probability of the default metafounder is not 0.5.

Reply:
Ahh, I had missed that bit. Thank you!

Reply:
@XingerTang @RosCraddock I am going over this PR discussion trying to bring it to a merge, but on this part I am a bit confused what is going on. Can you please help me understand this:

>  n the tinyhouse.Pedigree, I set metafounders for all of the individuals for simplicity, so the progenies of the progenies of different metafounders all share a default metafounder, so we don't want to update their anterior probabilities otherwise it would interfere with the information while passing down the pedigree if the alternative allele probability of the default metafounder is not 0.5.

Reply:
I can see in Tinyhouse that, by default, all individuals in the pedigree are assigned a metafounder. This is either the default (MF_1) or the main_metafounder defined by the user. The founders' metafounder is later updated to the one they have been allocated per the pedigree input. Thus, to update the anterior probability of the founders, we must use the metafounder and check the individual is a founder with `isFounder()`. Otherwise, we could unintentionally update the anterior probabilities of the progeny with the allele frequency of the main metafounder (i.e the default of 0.5). This would be unnecessary and reduce computational efficiency. However, I don't know how this impacts peeling since the anterior probabilities of the progeny are later informed by the parents. @XingerTang could you clarify your last sentence in the above [comment](https://github.com/AlphaGenes/AlphaPeel/pull/175#discussion_r1745131168)?

Reply:
@RosCraddock is right that most of the time, the anterior probabilities of the progenies are recalculated and overwritten using the probabilities from the parents during the peeling, so it would only lead to the waste of time and computational power but not affect the result. It will cause a problem if we try to peel down the pedigree but not in the right order of generations, in this case, the anterior probabilities from the default metafounders would be used to calculate the genotype probabilities of all of the individuals and pass down or up. But these are all fixed with the conditioning with `isFounder()`.

Reply:
OK, I think all is well then.

Before we resolve this conversation and move onwards, I feel that this is an implementation detail that can easily get lost with time and could be a source of hard to figure out bugs. We should make a code comment somewhere! @XingerTang @RosCraddock please work on this comment as part of this PR. Thanks!

Reply:
@gregorgorjanc That's a good point! I will add comments in tinyhouse and tinypeel.py to clarify this.

Reply:
@RosCraddock thanks for doing this at https://github.com/AlphaGenes/tinyhouse/pull/19